src/lib.rs

```diff
@@ -159,12 +163,20 @@ impl EguiClipboard {
 
     /// Gets clipboard contents. Returns [`None`] if clipboard provider is unavailable or returns an error.
     #[must_use]
+    #[cfg(not(target_arch = "wasm32"))]
     pub fn get_contents(&self) -> Option<String> {
         self.get_contents_impl()
     }
 
+    /// Gets clipboard contents. Returns [`None`] if clipboard provider is unavailable or returns an error.
+    #[must_use]
+    #[cfg(target_arch = "wasm32")]
+    pub fn get_contents(&mut self) -> Option<String> {
+        self.get_contents_impl()
+    }
+
     #[cfg(not(target_arch = "wasm32"))]
-    fn set_contents_impl(&self, contents: &str) {
+    fn set_contents_impl(&mut self, contents: &str) {
```

Comment:
I'm not sure I intended that ? but it's probably weird to have different mutability depending on wasm on not

Reply:
Yeah, we definitely should have a uniform interface. Let's keep everything `&mut self` then?